src/python/pants/backend/javascript/dependency_inference/rules.py

```diff
@@ -135,17 +135,13 @@ def _create_inference_metadata(
     )
 
 
-async def _prepare_inference_metadata(address: Address, file_path: str) -> InferenceMetadata:
-    owning_pkg, maybe_config = await concurrently(
-        find_owning_package(OwningNodePackageRequest(address)),
-        find_parent_ts_config(ParentTSConfigRequest(file_path, "jsconfig.json"), **implicitly()),
-    )
+async def _prepare_inference_metadata(owning_pkg, maybe_config, spec_path) -> InferenceMetadata:
```

Comment:
It'd be good to ensure there's type annotations on these args if possible.

Reply:
Ah yep, meant to get back around to that and never did...